SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestCharacterMovement/SpatialTestCharacterMovement.cpp

```diff
@@ -50,110 +50,108 @@ void ASpatialTestCharacterMovement::BeginPlay()
 {
 	Super::BeginPlay();
 
-	// Universal setup step to create the TriggerBox and to set the 2 helper variables
-	AddUniversalStep(TEXT("UniversalSetupStep"), nullptr, [](ASpatialFunctionalTest* NetTest) {
-		ASpatialTestCharacterMovement* Test = Cast<ASpatialTestCharacterMovement>(NetTest);
-		Test->bCharacterReachedDestination = false;
-		Test->ElapsedTime = 0.0f;
-
-		ATriggerBox* TriggerBox = Test->GetWorld()->SpawnActor<ATriggerBox>(FVector(232.0f, 0.0f, 40.0f), FRotator::ZeroRotator, FActorSpawnParameters());
-
-		UBoxComponent* BoxComponent = Cast<UBoxComponent>(TriggerBox->GetCollisionComponent());
-		if (BoxComponent)
+	// Universal setup step to create the TriggerBox and to set the helper variable
+	AddUniversalStep(TEXT("UniversalSetupStep"), nullptr, [this](ASpatialFunctionalTest* NetTest)
```

Comment:
Why is this a universal step? Wouldn't a server step be sufficient?

Reply:
It could be a server step,but that would then require setting the `bCharacterReachedDestination `to false in either another universal step, or after asserting that it is true, in order to allow the test to be ran multiple times. I think it might be beneficial since it would then also allow to register the TriggerBox for auto destroy and therefore remove the last server step, will do!